frontend/src/components/login/LoginModal/LoginModal.jsx

```diff
@@ -161,10 +164,13 @@ const LoginModal = ({ toggleLogin, toggleRegister }) => {
             </div>
             <div className={styles.googleLogin}>
               <GoogleLogin
-                handleGoogleLogin={(token) => {
-                  handleGoogleLogin(token);
-                  toggleLogin();
-                }}
+                handleGoogleLogin={useCallback(
```

Comment:
Maybe it's better to use a variable here for clarity and consistency

Reply:
done